mlos_bench/mlos_bench/tests/tunables/tunables_copy_test.py

```diff
@@ -6,18 +6,21 @@
 Unit tests for deep copy of tunable objects and groups.
 """
 
+from mlos_bench.tunables.tunable import Tunable
+from mlos_bench.tunables.tunable_groups import TunableGroups
 
-def test_copy_tunable_int(tunable_int):
+
+def test_copy_tunable_int(tunable_int: Tunable) -> None:
     """
     Check if deep copy works for Tunable object.
     """
     tunable_copy = tunable_int.copy()
     assert tunable_int == tunable_copy
-    tunable_copy.value += 200
+    tunable_copy.value = tunable_copy.numerical_value + 200
```

Comment:
Since the otherside could technically be `Union[int, float, str]` mypy can't figure out what `+=` means, which is understandable.
Might be able to overload `__add__` or some such, but I didn't bother since this only shows up in one spot so far.

Reply:
nah, that line looks great. let's keep it like this

Reply:
It's a little funny I think because we reference `numerical_value` property and assign `value` which is a convenience cast function essentially, but feels like it leaks too much internal details.
Anyways, I may go back and fix it later, but for now I think it's good enough.